operations/power_management/Cray_Advanced_Platform_Monitoring_and_Control_CAPMC.md

```diff
@@ -1,172 +1,119 @@
-```
-
-**Power Off a Chassis 0 and Its Descendents**
-
-```bash
-ncn# cray capmc xname_off create --xnames x1000c0 --recursive --format json
-```
-
-**Power Off Node 0 in Cabinet 1000, Chassis, 0, Slot 0, Node Card 0**
-
-```bash
-ncn# cray capmc xname_off create --xnames x1000c0s0b0n0 --format json
-```
-
-**Emergency Power Off \(EPO\) CLI Command**
-
-```bash
-ncn# cray capmc emergency_power_off –-xnames LIST_OF_CHASSIS --force --format json
-```
-
-To recover or "reset" the components after a software EPO, set the chassis to a known hardware state \(off\). The cabinet\(s\) can then be powered on normally after the EPO is cleared. For a complete procedure, see [Recover from a Liquid Cooled Cabinet EPO Event](Recover_from_a_Liquid_Cooled_Cabinet_EPO_Event.md).
-
-```bash
-ncn# cray capmc xname_off create --xnames LIST_OF_CHASSIS --force true
-e = 0
-err_msg = ""
-```
-
+- Chassis
+- Slingshot Switch modules
+- Compute blades
+- Compute nodes
+
+CAPMC uses xnames to specify entire cabinets or specific components throughout
+the system. By default, CAPMC controls power to only one component at a time. A
+`--recursive true` option can be passed to CAPMC using the `cray` CLI. When the
+`--recursive true` option is included in a request, all of the sub-components of
+the target component are included in the power command.
+
+By the cabinet naming convention, each cabinet in the system is assigned a
+unique number. Cabinet numbers can range from 0-9999 and contain from 1-4 digits
+only.
+
+Although manufacturing typically follows a sequential cabinet numbering scheme:
+- Liquid Cooled cabinet numbers: x1000–x2999
+- Air Cooled cabinet numbers: x3000–x4999
+- Liquid Cooled TDS cabinet numbers: x5000–5999
+
+**Examples of valid xnames:**
+- Full system: s0, all
+- Cabinet numbers: x1000, x3000, x5000
+- Chassis numbers 0-7: x1000c7, x3500c0 (Air Cooled cabinets are always
+chassis 0)
+- Compute Blade Slots 0-7: x1000c7s3, x3500c0s15 (U15)
+- Compute Nodes: x1000c7s3b0n0, x3500c0s15b1n0
+- NCN Slots: x3200c0s9 (U9)
+- NCN Nodes: x3200c0s9b0n0
+
+## Power capping
+- get_power_cap_capabilities
+- get_power_cap
+- set_power_cap
+
+CAPMC is capable of setting node power limits on all supported compute node
+hardware in both liquid cooled cabinets and air cooled cabinets. This
+functionality enables external software to establish an upper bound, or estimate
+a minimum bound, on the amount of power a system may consume. Separate CAPMC
+calls are required to power cap different compute node types as each compute
+node type has its own power capping capabilities.
+
+**NOTE:** Power capping is not supported for liquid cooled chassis, switch
+modules, compute blades, and any non-compute nodes (NCNs) in air cooled
+cabinets.
+
+## Deprecated interfaces
+[CAPMC Deprecation Notice](../../introduction/CAPMC_deprecation.md)
+
+### Power control and query by NID
+Use the interfaces from [Power control and query by xname](#power-control-and-query-by-xname)
```

Comment:
The first line under each of these headings should have a `:` character at the end of the line.